samples/test/datasets.test.js

```diff
@@ -174,16 +174,22 @@ describe('Datasets', () => {
     );
 
     for (const dataset of datasets) {
-      const [metadata] = await dataset.getMetadata();
-      const creationTime = Number(metadata.creationTime);
-
-      if (isResourceStale(creationTime)) {
-        try {
+      let metadata
+      try {
+        [metadata] = await dataset.getMetadata();
+      } catch (e) {
+        console.log(`dataset(${dataset.id}).getMetadata() failed`);
+        console.log(e);
+        return;
+      }
+      try {
```

Comment:
let's leave the original try/catch at this original place since it's only supposed to catch errors when trying to delete, we don't want to suppress valid errors in case any occurs

Reply:
I was doing so right before this comment 😂  just pushed that